### ISSUE TITLE
MailWatch-DB.pm

### ISSUE THREAD

**Issue by stefaweb** [mailwatch/1.2.0#581]
No longer works.

`Feb 23 17:30:25 antispam-dev MailScanner[3339]: Could not use Custom Function code MailScanner::CustomConfig::InitSQLWhitelist, it could not be "eval"ed. Make sure the module is correct with perl -wc (Error: DBI connect('database=;host=','',...) failed: Access denied for user 'Debian-exim'@'localhost' (using password: NO) at /usr/share/MailScanner/perl/custom/SQLSpamSettings.pm line 71.`

You forgot to update MailWatch-DB.pm.

```
sub mailwatch_get_db_name { return $db_name };
sub mailwatch_get_db_host { return $db_host };
sub mailwatch_get_db_user { return $db_user };
sub mailwatch_get_db_password { return $db_pass };

```
Is like this in others .pm files.
```
my ($db_name) = mailwatch_get_db_name();
my ($db_host) = mailwatch_get_db_host();
my ($db_user) = mailwatch_get_db_user();
my ($db_pass) = mailwatch_get_db_password();
```

**Comment by asuweb** [comment]
It works on mine

**Comment by stefaweb** [comment]
Sorry the issue title was wrong.

On develop, MailWatch-DB.pm have the old variable name without mailwatch_

**Comment by stefaweb** [comment]
Done.

**Comment by Skywalker-11** [comment]
Is it working for with current develop? On a fresh install MailWatch.pm does not load the credentials and so fails. It works if I rename MailWatch-DB.pm to 0MailWatch-DB.pm

**Comment by stefaweb** [comment]
Work on my dev-server without changing the name. I'm using MS 5.0.4-4. I'm also using the fix #597 to try.

**Comment by stefaweb** [comment]
We can may be change the name for "00-mailwatch-conf.pm" to be sure.
